components/brave_origin/brave_origin_policy_manager.cc

```diff
@@ -78,6 +70,16 @@ std::optional<bool> BraveOriginPolicyManager::GetPolicyValue(
   return GetPolicyValueInternal(*policy_info, policies_dict, profile_id);
 }
 
+bool BraveOriginPolicyManager::IsBrowserPolicy(
+    std::string_view pref_name) const {
+  return base::FindOrNull(browser_policy_definitions_, pref_name) != nullptr;
```

Comment:
Could this be `browser_policy_definitions_.contains(pref_name)`? This also applies to the method below